scripts/install_third_party_libs.py

```diff
@@ -157,11 +152,11 @@ def install_buf_and_protoc():
         os.remove(os.path.join(BUF_DIR, protoc_file))
     except Exception as e:
         raise Exception('Error installing protoc binary') from e
-    common.recursive_chmod(buf_path, 0o744)
-    common.recursive_chmod(protoc_path, 0o744)
+    common.recursive_chmod(buf_path, 0o744)  # type: ignore[no-untyped-call]
+    common.recursive_chmod(protoc_path, 0o744)  # type: ignore[no-untyped-call]
 
 
-def compile_protobuf_files(proto_files_paths):
+def compile_protobuf_files(filepaths: List[str]) -> None:
```

Comment:
Why is the name changed?

Reply:
I figured the "proto" part of the name proto_files_paths was redundant, as the method is already named compile_protobuf_files; that would just leave us with files_paths, but I've generally seen 'file path' written as one word, filepath (for instance, [here](https://docs.python.org/3/library/os.path.html#os.path.relpath) in the Python os.path.relpath documentation. That being said, if you think I should change it back, I will.